tests/attention/test_mla_cuda_graph_planning.py

```
@@ -0,0 +1,644 @@
+            return backend
+
+    _patch_plan_from_wrapper_owner(monkeypatch, backend_name, _Backend)
+    wrapper = mla_core.BatchMLAPagedAttentionWrapper(
+        torch.empty(1), use_cuda_graph=True, backend=backend_name
+    )
+    args = _dense_plan_args(backend_name)
+
+    wrapper.plan(**args)
+    assert wrapper._backend_impl is backend
+
+    with pytest.raises(ValueError, match="CUDA graph.*replan"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)